src/components/paragraphs/sum-top-banner/sum-top-banner-paragraph.tsx

```diff
@@ -17,47 +17,61 @@ type Props = HtmlHTMLAttributes<HTMLDivElement> & {
 
 const SumTopBannerParagraph = ({paragraph, pageTitle, ...props}: Props) => {
   const behaviors = getParagraphBehaviors(paragraph)
+  const bgColor = behaviors.sum_banner_behaviors?.sum_banner_overlay_bkg || "olive"
 
   return (
     <div
       {...props}
       className={twMerge("mb-32", props.className)}
     >
       <HeroBanner
-        className="mb-0"
+        cardColorClasses={clsx("mb-0 border border-white border-b-0 border-x-0 border-b-4 lg:border-x-4 lg:border-y-0 bg-poppy-light",
+          {
+          "bg-olive-light": bgColor === "olive",
+          "bg-spirited-light": bgColor === "spirited",
+          "bg-spirited-dark": bgColor === "spirited_dark",
+          }
+        )}
         imageUrl={paragraph.sumTopBannerImage?.mediaImage.url}
         imageAlt={paragraph.sumTopBannerImage?.mediaImage.alt}
         overlayPosition={behaviors.hero_pattern?.overlay_position}
```

Comment:
This should be `behaviors?.sum_top_banner_behavior?.sum_top_banner_alignment || "right"`